README.md

```diff
@@ -10,37 +10,51 @@ To get started set up an account on [Kinde](https://app.kinde.com/register).
 
 Clone this repo and install dependencies by running `mvn clean install`
 
-In `application.properties` file, set the following variables with the details from the Kinde `App Keys` page
-
-> kinde.host - The token host value
->
-> kinde.client.secret - The client secret
-
-e.g
-
-```
-kinde.host=https://<your_subdomain>.kinde.com
-kinde.client.secret=<your_client_secret>
+## Project Overview
+This project consists of the following sub projects
+- kinde-core-example: This is a unit test that provides an example of how to use the library directly.
+- kinde-j2ee-exmaple: An example J2EE project, using the Kinde-j2ee servlet.
+- kinde-management-example: This is a unit test that gives you an example of how to use the management API directly.
+- springboot-pkce-client-example: A standalone pkce springboot example.
+
+## Configuring the kinde-core-example and kinde-management-example
+In `.env` file, set the following variables with the details from the Kinde `App Keys` page
+
+```shell
+KINDE_DOMAIN=https://burntjam.kinde.com
+KINDE_CLIENT_ID=<replace>
+KINDE_CLIENT_SECRET=<replace>
+KINDE_REDIRECT_URI=http://localhost:8080/kinde-j2ee-app/login
+KINDE_GRANT_TYPE=CODE
+KINDE_SCOPES=openid
 ```
 
-## Set your Callback and Logout URLs
-
-Your user will be redirected to Kinde to authenticate. After they have logged in or registered they will be redirected back to your Java application.
-
-You need to specify in Kinde which url you would like your user to be redirected to in order to authenticate your app.
-
-On the App Keys page set ` Allowed callback URLs` to `http://localhost:8080/api/auth/kinde_callback`
+## Configurating the kinde-j2ee-example
+The kinde-j2ee-example is a war, and is deployed into a running application server such as tomcat. In order to configure it, you need to export the following variables

```

Comment:
Is the example Tomcat (you say "such as Tomcat")? How does one deploy this into Tomcat?

Reply:
Applied the changes